core/templates/pages/exploration-editor-page/services/exploration-property.service.ts

```diff
@@ -90,7 +90,7 @@ export class ExplorationPropertyService {
   // Validates the given value and returns a boolean stating whether it
   // is valid or not. THIS CAN BE OVERRIDDEN BY SUBCLASSES. The default
   // behavior is to always return true.
-  _isValid(value: string | boolean): boolean {
+  _isValid(value: string | boolean): string | boolean {
```

Comment:
Why is the return value allowed to be a string here? Or should the comment above this be changed as well?

Reply:
done